### PR TITLE
batch of fixes for the controller.

### PR DESCRIPTION
**Pull request by morvencao**
- remove unused resource watch
- add more debug info

### CODE REVIEW

File: pkg/helmreconciler/listeners.go [hunk 1]
```diff
@@ -258,25 +258,41 @@ func (l *LoggingRenderingListener) BeginResource(obj runtime.Object) (runtime.Ob
 
 // ResourceCreated logs the event
 func (l *LoggingRenderingListener) ResourceCreated(created runtime.Object) error {
-	log.Info("new resource created")
+	accessor := meta.NewAccessor()
+	kind, _ := accessor.Kind(created)
+	name, _ := accessor.Name(created)
+	version, _ := accessor.APIVersion(created)
+	log.Infof("resource created (%s, %s, %s)", version, kind, name)
```
Comment (ayj) [inline pkg/helmreconciler/listeners.go]: You can get the full group/version/kind from the runtime object directly, i.e. ```created.GetObjectKind().GroupVersionKind()```

File: pkg/controller/istiocontrolplane/istiocontrolplane_controller.go [hunk 2]
```diff
@@ -73,13 +72,13 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
...
-	if err != nil {
-		return err
-	}
+	//err = c.Watch(&source.Kind{Type: &corev1.Pod{}}, &handler.EnqueueRequestForOwner{
+	//        IsController: true,
+	//        OwnerType:    &v1alpha2.IstioControlPlane{},
+	//})
+	//if err != nil {
+	//        return err
+	//}
```
Comment (ostromart) [inline pkg/controller/istiocontrolplane/istiocontrolplane_controller.go]: I think you can just remove this. I don't see a use case for us wanting to watch Pods.

Comment (morvencao) [reply pkg/controller/istiocontrolplane/istiocontrolplane_controller.go]: removed this section.

File: pkg/helmreconciler/listeners.go (hunk 1 above)
Comment (morvencao) [reply pkg/helmreconciler/listeners.go]: Thanks, done.

**Review by ostromart — APPROVED** [review]

**Pull request merged**